src/components/presentational/Layout/Layout.css

```diff
@@ -1,5 +1,11 @@
 ﻿.mdhui-layout {
-    padding-bottom: env(safe-area-inset-bottom);
```

Comment:
Hmm is removing this safe?  IIRC this was added for a reason on iOS

Reply:
(Though it seems fine in my testing, so IDK)

Reply:
If you look at `Layout.tsx` there's `let paddingBottom = props.flex ? "0" : "env(safe-area-inset-bottom)";` so to clarify I did not completely remove this, just made it conditional on the `Layout` not being flex.

Reply:
got it